src/mame/midway/seattle.cpp

```diff
@@ -2844,30 +2845,29 @@ ROM_START( hyprdriv )
 
 	*/
 
-	ROM_REGION32_LE( 0x80000, PCI_ID_GALILEO":rom", 0 )
-	ROM_LOAD( "hyperdrive1.1.u32", 0x000000, 0x80000, CRC(3120991e) SHA1(8e47888a5a23c9d3c0d0c64497e1cfb4e46c2cd6) )  // Boot Rom Version 2. Doesn't work, maybe for older drive?
-	ROM_LOAD( "hyprdrve.u32", 0x000000, 0x80000, CRC(3e18cb80) SHA1(b18cc4253090ee1d65d72a7ec0c426ed08c4f238) )  // Boot Rom Version 9.
-
 	DISK_REGION( PCI_ID_IDE":ide:0:hdd" ) // Version 1.40  Oct 23 1998  15:16:00
 	DISK_IMAGE( "hyprdriv", 0, SHA1(8cfa343797575b32f46cc24150024be48963a03e) )
+
+	ROM_REGION16_LE( 0x10000, "dcs", 0 ) // ADSP-2115 data Version 1.02
+	ROM_LOAD16_BYTE( "seattle.snd", 0x000000, 0x8000, BAD_DUMP CRC(bec7d3ae) SHA1(db80aa4a645804a4574b07b9f34dec6b6b64190d) )	
```

Comment:
Anyone knows why this is marked `BAD_DUMP` if it matches the dcs roms of (at least) all the other sets?

Reply:
Quite frankly, there is no reason anymore after performing a sound subsystem test on hardware and mame. The numbers on both tests match meaning there is no reason to keep the flag.